javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/NosqlInjectionATM.qll

```diff
@@ -9,63 +10,20 @@ private import semmle.javascript.heuristics.SyntacticHeuristics
 private import semmle.javascript.security.dataflow.NosqlInjectionCustomizations
 import AdaptiveThreatModeling
 
-class NosqlInjectionAtmConfig extends AtmConfig {
-  NosqlInjectionAtmConfig() { this = "NosqlInjectionATMConfig" }
+class Configuration extends AtmConfig {
+  Configuration() { this = "NosqlInjectionATMConfig" }
```

Comment:
I tried to follow the convention on configurations as I understood it:  The existing code was
```
class Configuration extends TaintTracking::Configuration {
  Configuration() { this = "NosqlInjectionATM" }
```
One level up, in TaintTracking.qll, it also reads
```
abstract class Configuration extends DataFlow::Configuration
```
I understood this to mean that for some reason the convention is that configuration classes are always called `Configuration`, rather than something more specific such as `NosqlInjectionATMConfig`. I don't particularly like this either, so if there's no reason to do it this way I'm happy to change it!

Reply:
@adityasharad in the interest of getting an answer today, do you know whether such a convention exists? If not, I'll make the suggested change and push a commit so @kaeluka can re-approve the PR tomorrow.

Reply:
I don't think it's a strict convention; you see both in the libraries across languages. I think it's fine to keep this diff smaller and keep the string and class names in sync.

Reply:
Perfect -- thank you!

Reply:
Fixed in c5184d37e7c8b3cfafaf5dcafab850589315bfac -- I do think this convention is nicer! Note that I changed e.g. `NosqlInjectionATMConfig` to `NosqlInjectionAtmConfig` to fit the style guide that acronyms should have only the first letter capitalized (e.g. `Http` rather than `HTTP`).